usersync/usersyncers/syncer_test.go

```diff
@@ -99,6 +99,7 @@ func TestNewSyncerMap(t *testing.T) {
 		openrtb_ext.BidderInMobi:       true,
 		openrtb_ext.BidderKidoz:        true,
 		openrtb_ext.BidderKubient:      true,
+		openrtb_ext.BidderMobfoxpb:     true,
```

Comment:
Just wanna confirm that your adapter doesn't support usersync at this time, correct?

Reply:
Yes, we are currently working on it